status/projects.py

```diff
@@ -278,6 +286,12 @@ class ProjectsSearchHandler(ProjectsBaseDataHandler):
     Loaded through /api/v1/project_search/([^/]*)$
     """
     def get(self, search_string):
+        # Searching for P1234 doesn't work, but 1234 does.
+        # If I'm honest, I don't know why. But let's just strip the P
+        # and get on with enjoying life.
```

Comment:
Change this [line](https://github.com/SciLifeLab/genomics-status/pull/212/files#diff-04bc69ae72172927a34c586ac659931cL239) to:
`if search_string.lower() in row.value['project_name'].lower() or search_string.lower() in row.key[1].lower():`

I'd make a PR, but I'm too lazy